.github/workflows/build.yml

```diff
@@ -0,0 +1,17 @@
+name: 'ecma-402'
+
+on: [pull_request, push]
+
+jobs:
+  build:
+    name: 'build & lint ecmarkup'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ljharb/actions/node/run@125357c
```

Comment:
@ljharb this is erroring, trying to install npm or something. Is there a quick link to this action code?

Reply:
removing the npm ci config should have resolved this.

https://github.com/ljharb/actions tho